src/System.Text.Encoding/tests/Fallback/EncoderReplacementFallbackTests.cs

```diff
@@ -109,7 +109,8 @@ public void CreateFallbackBuffer_Fallback_InvalidSurrogateChars_ThrowsArgumentOu
             EncoderFallbackBuffer buffer = new EncoderReplacementFallback().CreateFallbackBuffer();
 
             Assert.Throws<ArgumentOutOfRangeException>("charUnknownHigh", () => buffer.Fallback('a', '\uDC00', 0));
-            Assert.Throws<ArgumentOutOfRangeException>("CharUnknownLow", () => buffer.Fallback('\uD800', 'a', 0));
+            Assert.Throws<ArgumentOutOfRangeException>(
+                RuntimeDetection.IsNetFramework ? "CharUnknownLow" : "charUnknownLow", () => buffer.Fallback('\uD800', 'a', 0));
```

Comment:
@Clockwork-Muse  thanks for the clarification. I would suggest you close this PR till the other coreclr changes get merged and the nuget package get built and referenced in corefx. you have to wait for that before we can approve the changes here.